static/temp/foods.json

```diff
@@ -0,0 +1,332 @@
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        }
+      ]
+    },
+    "images": ["/img1", "/img2", "/img3"],
+    "nutrition": {
+      "availableAmounts": ["1 tablespoon", "1cup", "100g"],
+      "source": "USDA",
+      "dashboard": [
+        { "name": "Protein", "percentage": "10%", "weight": "120g" },
+        { "name": "Carbs", "percentage": "20%", "weight": "120g" },
+        { "name": "Fats", "percentage": "30%", "weight": "120g" },
+        { "name": "Fiber", "percentage": "40%", "weight": "120g" }
+      ],
+      "nutrients": ["Protein", "Carbohydrate", "Water", "Vitamins", "Fats", "Fiber", "Minerals"],
+      "allFilters": {
+        "lifeStages": [
+          {
+            "name": "Infant",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Children",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          },
+          {
+            "name": "Male",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Female",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          },
+          {
+            "name": "Pregnant",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Location",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          }
+        ],
+        "ages": [
+          {
+            "start": 9,
+            "startUnit": "years",
+            "end": 13,
+            "endUnit": "years"
+          },
+          {
+            "start": 14,
+            "startUnit": "years",
+            "end": 18,
+            "endUnit": "years"
+          },
+          {
+            "start": 19,
+            "startUnit": "years",
+            "end": 30,
+            "endUnit": "years"
+          },
+          {
+            "start": 31,
+            "startUnit": "years",
+            "end": 50,
+            "endUnit": "years"
+          },
+          {
+            "start": 51,
+            "startUnit": "years",
+            "end": 70,
+            "endUnit": "years"
+          }
+        ]
+      },
+      "tableHeaders": {
+        "allNutrients": [
+          "Protein",
+          "Carbohydrate",
+          "Water",
+          "Vitamin",
+          "Fats",
+          "Fiber",
+          "Minerals"
+        ],
+        "valuePercent": "% Daily Value",
+        "rdi": {
+          "amount": 130,
+          "unit": "g"
+        }
+      },
+      "tableContent": [
+        {
+          "title": "Total Carbohydrate",
+          "amount": 4,
+          "amountUnit": "g",
+          "content": [
+            {
+              "nutrient": "Carbohydrate  2g",
```

Comment:
This is not a correct representation. It mixes nutrient namd with value amount, and unit. Break this into following

```json5
{
  "nutrient": "Carbohydrate",
  "value": {
    "amount": 2,
    "unit": "g",
  }
}
```

Do this for other `nutrient` in `content` block.

Reply:
done